src/marker-parser.js

```diff
@@ -176,30 +185,41 @@ export default class MarkerParser {
          * Groups:
          *     1: The tag id
          */
-        this._endTagDecodeRegExp = new RegExp(`^(\\S+)\\s*$`);
+        this._endTagDecodeRegExp = new RegExp(`^(\\S+)\\s*(\\S*)?$`);
```

Comment:
(The `?` doesn't do anything here; you could remove it if you wanted.)

Reply:
Oh, yeah, good point.